guava-testlib/src/com/google/common/testing/ArbitraryInstances.java

```diff
@@ -393,7 +393,7 @@ private static MatchResult newMatchResult() {
     } catch (IllegalAccessException impossible) {
       throw new AssertionError(impossible);
     } catch (InvocationTargetException e) {
-      logger.log(Level.WARNING, "Exception while invoking default constructor.", e.getCause());
+      logger.log(Level.SEVERE, "Exception while invoking default constructor.", e.getCause());
```

Comment:
This log level is inappropriate for this method. This method is attempting several ways to return an arbitrary instance, and while it's odd for a no-arg constructor to throw, it's not unacceptable on an arbitrary class.